src/coreclr/nativeaot/System.Private.TypeLoader/src/Internal/Runtime/TypeLoader/TypeLoaderEnvironment.Metadata.cs

```diff
@@ -102,6 +97,104 @@ internal static unsafe NativeReader GetNativeReaderForBlob(NativeFormatModuleInf
             return default(NativeReader);
         }
 
+        /// <summary>
```

Comment:
This is an addition - I copy-pasted the code from the hashtable in the deleted TypeLoaderEnvironment.NamedTypeLookup.cs. We were basically building a hashtable at runtime to cache things from this static hashtable.

This only makes sense if we can create brand new non-template types at runtime and is a waste of code and working set otherwise.